source/jormungandr/jormungandr/interfaces/v1/Places.py

```diff
@@ -199,6 +199,7 @@ def output(self, key, obj):
 
         lon, lat = get_lon_lat(obj)
         geocoding = obj.get('properties', {}).get('geocoding', {})
+        poi_type = geocoding.get('poi_types', [{}])[0]
```

Comment:
(hmmm, i'm not sure this is possible)
if `geocoding = {..., 'poi_types': [], ...}`,  
then `geocoding.get('poi_types', [{}])[0]` will crash...